tests/e2e/tests/test_notebook_images.py

```diff
@@ -23,11 +23,11 @@
 CUSTOM_RESOURCE_TEMPLATES_FOLDER = "./resources/custom-resource-templates"
 
 NOTEBOOK_IMAGES = [
-    "public.ecr.aws/j1r0q0g6/notebooks/notebook-servers/jupyter-scipy:v1.5.0",
-    "public.ecr.aws/c9e4w0g3/notebook-servers/jupyter-tensorflow:2.9.1-gpu-py39-cu112-ubuntu20.04-e3-v1.2",
-    "public.ecr.aws/c9e4w0g3/notebook-servers/jupyter-tensorflow:2.9.1-cpu-py39-ubuntu20.04-e3-v1.2",
-    "public.ecr.aws/c9e4w0g3/notebook-servers/jupyter-pytorch:1.12.0-gpu-py38-cu116-ubuntu20.04-ec2",
-    "public.ecr.aws/c9e4w0g3/notebook-servers/jupyter-pytorch:1.12.0-cpu-py38-ubuntu20.04-ec2",
+    "kubeflownotebookswg/jupyter-scipy:v1.6.0",
+    "public.ecr.aws/kubeflow-on-aws/notebook-servers/jupyter-tensorflow:2.9.1-gpu-py39-cu112-ubuntu20.04-e3-v1.2-2022-09-20",
```

Comment:
need to update the helm charts as well

https://github.com/awslabs/kubeflow-manifests/blob/main/charts/apps/jupyter-web-app/templates/ConfigMap/jupyter-web-app-config-tgcbb2gdkd-kubeflow-ConfigMap.yaml#L14-L19

Reply:
done